package.json

```diff
@@ -37,6 +37,7 @@
     "@storybook/addon-notes": "^5.3.19",
     "ember-cli-babel": "^7.18.0",
     "ember-cli-htmlbars": "^4.2.3",
+    "ember-cli-string-utils": "^1.1.0",
```

Comment:
Dans les blueprints on a pas accès aux "devDependencies" d'où l'ajout dans les dependencies ici 
(Voir ce passage là https://youtu.be/oebreLvpJWY?t=90)